cmd/krel/cmd/push.go

```diff
@@ -199,6 +199,7 @@ func runPushBuild(opts *pushBuildOptions) error {
 	}
 
 	if opts.versionSuffix != "" {
+		// TODO: what should we do with `latest` afterwards?
 		latest += "-" + opts.versionSuffix
```

Comment:
The golangci-lint target should throw an error here, which works locally but not inside the test infrastructure.